apps/templates/pages/login.html

```diff
@@ -75,9 +75,12 @@
             </div>
             <!-- /.col -->
             <div class="col-4">
-              <button type="submit" name="login" class="btn btn-primary btn-block">Sign In</button>
+              <button type="submit" name="login" class="btn btn-primary btn-block">Sign In</button>`
```

Comment:
Weird character in the end

```suggestion
              <button type="submit" name="login" class="btn btn-primary btn-block">Sign In</button>
```